bundle/src/files.rs

```diff
@@ -39,6 +40,8 @@ pub struct FileSet {
     pub file_set_type: FileSetType,
     pub files: Vec<BundledFile>,
     pub glob: String,
+    /// Added in v0.6.11. Populated when parsing from BEP, not from junit globs
+    pub test_runner_status: Option<JunitReportStatus>,
```

Comment:
I ended up making it optional again since that allows us to better deserialize. Otherwise we need to have some versioned logic here for deserializing from meta.json, which doesn't seem worth the effort for this particular case